openapi/SwarmDebug.yaml

```diff
@@ -1,6 +1,6 @@
-openapi: 3.0.0
+openapi: 3.0.3
```

Comment:
do they all have to have the same numbers? @janos

Reply:
not necessary to update, just checked and this is the current version of openapi